src/test/TestBMIAnalyzer.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *This file contains Junit test case for BMIAnalyzer.java
+ */
+
+package test;
+import main.java.BMIAnalyzer;
+import main.java.User;
+import org.junit.*;
+
+import java.util.HashMap;
+
+import static org.junit.Assert.*;
+public class TestBMIAnalyzer {
+    BMIAnalyzer bmi;
+    User user;
+
+    @Before
+    public void setUp() throws Exception {
+        bmi = new BMIAnalyzer();
+        HashMap<String, Object> userInfo = new HashMap<String, Object>();
+
+        userInfo.put("height", "1.70");
+        userInfo.put("weight", "58");
+        userInfo.put("age", "21");
+        user = new User("Amy", "F", userInfo);
+
+    }
+
+    @Test(timeout = 500)
+     public void testOutput() {
```

Comment:
One test case should be fine here, but are there any possible cases that we can test? What if one of the variables Height/Weight is zero? This will cause an ArithmeticException (Divide by zero) and what will we do with that? Restrict such that the height and the weight can't be zero when the user is inputting it?